api/FileReader.json

```diff
@@ -563,7 +563,8 @@
               "version_added": "32"
             },
             "ie": {
-              "version_added": "10"
+              "version_added": "10",
+              "version_removed": "11"
```

Comment:
Looks like this was never supported at all. I'd go with only `"version_added": false` and no `"version_removed"` line instead

Reply:
done